plasmapy/physics/tests/test_parameters.py

```diff
@@ -443,17 +443,24 @@ def test_gyrofrequency():
 
     assert gyrofrequency(B).unit.is_equivalent(u.rad / u.s)
 
+    assert gyrofrequency(B, to_hz=True).unit.is_equivalent(u.Hz)
+
     assert np.isclose(gyrofrequency(1 * u.T).value, 175882008784.72018)
 
     assert np.isclose(gyrofrequency(2.4 * u.T).value,
                       422116821083.3284)
 
+    assert np.isclose(gyrofrequency(1 * u.T, to_hz=True).value,
+                      27992490076.528206)
+
     assert np.isclose(gyrofrequency(2.4 * u.T, signed=True).value,
                       -422116821083.3284)
 
     assert np.isclose(gyrofrequency(1 * u.G).cgs.value,
                       1.76e7, rtol=1e-3)
 
+    assert np.isclose(gyrofrequency())
+
```

Comment:
[This test is faling](https://dev.azure.com/plasmapy/PlasmaPy/_build/results?buildId=122), as it probably should be without any arguments being passed in!

Reply:
my bad, I forgot to enter values in it. fixing it now